src/collections/_documentation/workflow/integrations/global-integrations.md

```diff
@@ -177,6 +177,98 @@ Fixes MYAPP-317
 
 When Sentry sees this, we’ll automatically annotate the matching issue with a reference to the commit, and, later, when that commit is part of a release, we’ll mark the issue as resolved.
 
+
+### Bitbucket Server
+
+You can now use the data from your Bitbucket Server commits to help you find and fix bugs faster.
+
+#### Installing Bitbucket Server with Sentry
+
+{% capture __alert_content -%}
+Sentry owner or manager permissions, and Bitbucket administrator permissions are required to install this integration.
+{%- endcapture -%}
+{%- include components/alert.html
+  title="Note"
+  content=__alert_content
+  level="warning"
+%}
+
+##### I. Generate an RSA public/private key pair
+To generate an RSA public/private key pair, run the following commands in your terminal window one by one.
+```
+openssl genrsa -out bitbucket_privatekey.pem 1024
+openssl req -newkey rsa:1024 -x509 -key bitbucket_privatekey.pem -out bitbucket_publickey.cer -days 365
+openssl pkcs8 -topk8 -nocrypt -in bitbucket_privatekey.pem -out bitbucket_privatekey.pcks8
+openssl x509 -pubkey -noout -in bitbucket_publickey.cer  > bitbucket_publickey.pem
+```
+
+##### II. Create a new application link in Bitbucket
+1. In Bitbucket, click the **gear icon** > **Applications** > **Application Links**.
+1. Enter the following as the application URL: 
+`https://sentry.io/extensions/bitbucket_server/setup/`
+1. Click **Create New Link**. If you see a warning that “No response was received from the URL you entered”, ignore and click **Continue**.
+1. In the resulting dialog, fill out the form as follows:
+
+    | Application Name                 | Sentry        |
+    | Application Type                   | Generic Application |
+    | Service Provider Name | Sentry |
+    | Consumer Key                       | (your choice, but keep this handy for the next step) |
+    | Shared Secret                     | sentry |
+    | Request Token URL                  | https://sentry.io |
+    | Access Token URL | https://sentry.io    |
+    | Authorize URL           | https://sentry.io    |
+    | Create Incoming Link               | No    |
+1. Click **Continue**. You should be returned to the **Configure Application Links** page, where you should see an application called **Sentry**.
```

Comment:
```suggestion
1. Click **Continue**. This will return you to the **Configure Application Links** page, where you'll see an application called **Sentry**.
```